service-request-infno-spec-1.0.0-swagger.json

```diff
@@ -0,0 +1,213 @@
+{
+  "openapi" : "3.0.0",
+  "servers" : [ {
+    "url" : "https://qaeservices1.capetown.gov.za/coct/api"
+  } ],
+  "info" : {
+    "description" : "This is a example API for Service Request.",
```

Comment:
`"This is a radically open, non-authed API intended to provide information to the residents of Cape Town"`

Reply:
Description has been updated